app/src/main/java/com/m/androidNativeApp/MainActivity.java

```diff
@@ -142,33 +156,40 @@ public void deleteTask(View view) {
 
         final String buttonId = button.getTag().toString();
 
+        AllTasksQuery tasksQuery = AllTasksQuery
+                .builder()
+                .build();
+        client.query(tasksQuery);
+
         DeleteTaskMutation deleteTask = DeleteTaskMutation
                 .builder()
                 .id(buttonId)
                 .build();
 
-        client.mutate(deleteTask).enqueue(new ApolloCall.Callback<DeleteTaskMutation.Data>() {
-            @Override
-            public void onResponse(@NotNull final Response<DeleteTaskMutation.Data> response) {
+        client.mutate(deleteTask)
+                .refetchQueries(tasksQuery)
```

Comment:
is calling `refetchQueries` here is just to update the cache?

Reply:
yes, every time there is delete task mutation we can update cache by calling refetch queries.